lang/jython/Portfile

```diff
@@ -67,13 +64,15 @@ if {![variant_isset installer]} {
     PortGroup       github 1.0
     github.setup    jythontools jython ${version} v
 
-    checksums       rmd160  34c48eae181694b590ce500175e452b83fdab2b8 \
-                    sha256  82a05fd72d8f681e99effa5dbfd148740ace444f226a3e479a5f3646f72af645 \
-                    size    46102901
+    checksums       rmd160  13ea84c24ffd0abbcf25a1b168d243ea6e57c86d \
+                    sha256  4938cc7079f48fa64e7558682b9bf9a62ebee16bbce698570007667945d08f60 \
+                    size    46106571
+    # Remove during next update
+    dist_subdir   ${name}/${version}_1
 
-    # Use Ant 1.10.x for Java 8 compatibility
+    # Use Ant 1.9.x for Java 7 compatibility
     depends_build-append \
-                    port:apache-ant
+                    port:apache-ant-1.9
```

Comment:
To ensure `jython -installer` builds with Java 7, Ant 1.9.x is needed. But I wonder if this would be somewhat annoying for those who normally build with Java ≥ 8 and Ant 1.10.x, as the the `apache-ant` and `apache-ant-1.9` ports cannot be installed simultaneously. One alternative is to drop Java 7 support (set `java.version 1.8+` and only need to use Ant 1.10.x). Another would be to drop the `-installer` variant entirely (even though some users have objected to MacPorts offering ports which can't be built from source).